elements/lisk-p2p/package-lock.json

```diff
@@ -146,23 +146,23 @@
 		},
 		"@liskhq/bignum": {
 			"version": "1.3.1",
-			"resolved": "https://registry.npmjs.org/@liskhq/bignum/-/bignum-1.3.1.tgz",
+			"resolved": "https://npm.lisk.io/@liskhq%2fbignum/-/bignum-1.3.1.tgz",
```

Comment:
it shouldnt update to the `npm.lisk.io`
probably u need to do  `npm config set registry https://registry.npmjs.com/` and re run the package.lock